api/src/test/java/org/pmiops/workbench/workspaces/WorkspacesControllerTest.java

```diff
@@ -2989,6 +2989,22 @@ public void testNotebookLockingMetadataUnknownUser() {
     assertNotebookLockingMetadata(gcsMetadata, expectedResponse, fcWorkspaceAcl);
   }
 
+  @Test
+  public void testGetBillingUsage() {
+    Workspace ws = createWorkspace();
+    ws = workspacesController.createWorkspace(ws).getBody();
+    stubGetWorkspace(ws.getNamespace(), ws.getId(), ws.getCreator(), WorkspaceAccessLevel.OWNER);
+    workspacesController.getBillingUsage(ws.getNamespace(), ws.getId());
```

Comment:
would we be able to extend this test case to verify the return value? I'm hoping its as simple as just saving a free tier row and asserting the return value from here.